test/scripts/helper.py

```diff
@@ -0,0 +1,19 @@
+from pathlib import Path
+import sys
+import importlib
+
+CWD = Path(__file__).parent.resolve()
+
+
+def import_module(module_name):
+    """Returns the module as object"""
+    # Get the right path to the module
+    path = str(CWD.parent.parent.joinpath('scripts', module_name))
+    spec = importlib.util.spec_from_loader(
+        module_name,
+        importlib.machinery.SourceFileLoader(module_name, path)
+    )
+    module = importlib.util.module_from_spec(spec)
+    spec.loader.exec_module(module)
+    sys.modules[module_name] = module
+    return module
```

Comment:
a helper to be able to import modules whose file suffix are not ending in `.py` like `valhalla_build_config` (with `valhalla_build_config.py` as filename we could just use `from ...scripts import valhalla_build_config`)